packages/browser-pool/package.json

```diff
@@ -22,7 +22,7 @@
 	"license": "Apache-2.0",
 	"repository": {
 		"type": "git",
-		"url": "git+https://github.com/apify/browser-pool-js"
+        "url": "https://github.com/apify/browser-pool.git"
```

Comment:
this should reflect the new repository, also version should be `3.0.0` as other packages